index.js

```diff
@@ -1,46 +1,85 @@
 'use strict'
 
+const fs = require('fs')
 const readPkg = require('read-pkg-up')
 const truncate = require('cli-truncate')
 const wrap = require('wrap-ansi')
 const pad = require('pad')
+const fuse = require('fuse.js')
+const homeDir = require('home-dir')
 const types = require('./lib/types')
 
-function getEmojiChoices(types) {
+const defaultConfig = {
+  types,
+  symbol: false
+}
+
+function getEmojiChoices({types, symbol}) {
   const maxNameLength = types.reduce(
     (maxLength, type) => (type.name.length > maxLength ? type.name.length : maxLength
   ), 0)
 
   return types.map(choice => ({
     name: `${pad(choice.name, maxNameLength)}  ${choice.emoji}  ${choice.description}`,
-    value: choice.code
+    value: symbol ? choice.emoji : choice.code,
+    code: choice.code
   }))
 }
 
+function loadConfig() {
+  return readPkg()
+    .then(({ pkg }) => {
+      const config = (pkg && pkg.config && pkg.config['cz-emoji'])
+      if (config) return config
+
+      return new Promise((resolve, reject) => {
+        fs.readFile(homeDir('.czrc'), 'utf8', (err, content) => {
+          if (err) reject(err)
+          const czrc = JSON.parse(content) || null
+          resolve((czrc && czrc.config && czrc.config['cz-emoji']) || {})
```

Comment:
Thanks for adding the `cz-emoji` section, here you can get rid off `|| {}` as `Object.assign` won't complain about `undefined` values.

Reply:
If you want to, you can also factorize things a bit:
```js
const getConfig = (obj) => obj && obj.config && obj.config['cz-emoji']
```